pkg/blobfuse-proxy/server/server.go

```diff
@@ -53,8 +63,15 @@ func (server *MountServer) MountAzureBlob(ctx context.Context,
 	authEnv := req.GetAuthEnv()
 	klog.V(2).Infof("received mount request: Mounting with args %v \n", args)
 
+	var cmd *exec.Cmd
 	var result mount_azure_blob.MountAzureBlobResponse
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
+	switch server.blobfuseVersion {
```

Comment:
BTW, I think in next PR, you could add "protocol: fuse2" in storage class, by that way, we could test blobfuse2 in our e2e test directly, and we could support blobfuse2 in 18.04 also.